src/builder/pod/mod.rs

```diff
@@ -281,12 +327,27 @@ impl PodBuilder {
     /// ```
     /// # use stackable_operator::builder::PodBuilder;
     /// # use stackable_operator::builder::ContainerBuilder;
+    /// # use k8s_openapi::{
+    ///     api::core::v1::ResourceRequirements,
+    ///     apimachinery::pkg::api::resource::Quantity
+    /// };
+    ///
     /// let pod = PodBuilder::new()
     ///     .metadata_default()
     ///     .add_container(
     ///         ContainerBuilder::new("container")
     ///             .unwrap()
     ///             .add_volume_mount("listener", "/path/to/volume")
+    ///             .resources(ResourceRequirements {
```

Comment:
Nit: Maybe we can have a fn on ContainerBuilder similar to `fn set_resources(cpu_limit: Quantity, memory_limit: Quantity, cpu_request:: Option<Quantity>, memory_request:: Option<Quantity>)`

Reply:
Agreed, will implement this.

Reply:
Added in d48d06c